config/commands/drupal.yml

```diff
@@ -31,4 +31,4 @@ command:
         options:
           sites-subdir: ${drupal.site.sites_subdir}
           settings-override-file: ${drupal.site.settings_override_file}
-          force-override-file: ${drupal.site.force_override_file}
+          force: ${drupal.site.force}
```

Comment:
We don't need this here since the default value is set directly in the method function signature to `false`.